app/controllers/signatures_controller.rb

```diff
@@ -19,6 +19,10 @@ def create
     end
   end
 
+  def signed
+    @petition = Petition.find(params[:petition_id])
```

Comment:
This method currently opens up all petitions from our system and it can be used to show a combination of any petition with any signature, since no checking that they relate is made. This is probably not very good thing.

The route is `petitions/:petition_id/signatures/:signature_id/signed`

So we can just do: 
`http://localhost:3000/petitions/100000/signatures/1/signed`

This is not a very big problem atm, since the page does not reveal any details of the signer. But if it did.. you could spread a link that is saying that someone signed a petition (even rejected one) they did not sign. We should not allow it.